### PR TITLE
Fix jenkinsfile

### PR DESCRIPTION
**Pull request by cosunae**
* Bug fix the main jenkisfile
* The same jenkinsfile should be used now for PR and main (using a jenkins multibranch project)
* Therefore we decomission the job DSL for the PR plans
* ignore requirement files for codespell, due to recent failures.

### CODE REVIEW

**Comment by cosunae** [comment]
launch jenkins

File: tmpl/jenkins/Jenkinsfile.j2 [hunk 1]
```diff
@@ -32,7 +32,6 @@ pipeline {
             steps {
                 sh '''#!/usr/bin/env bash
                 set -e
-                cd ${WORKSPACE}/{{ project_slug }}
```
Comment (twicki) [inline tmpl/jenkins/Jenkinsfile.j2:35]: we could leave the `cd ${WORKSPACE}` in if we want to be explicit about the directory we're in - but it should not be necessary

File: tmpl/README.md.j2 [hunk 2]
```diff
@@ -86,8 +86,8 @@ and in pull requests to the main branch.
 
 ### Jenkins
 
-Two jenkins plans are available in the `jenkins/` folder. On the one hand `jenkins/Jenkinsfile` controls the nightly (weekly, monthly, ...) builds, on the other hand
-`jenkins/JenkinsJobPR` controls the pipeline invoked with the command `launch jenkins` in pull requests on GitHub. Your jenkins pipeline will not be set up
+Two jenkins plans are available in the `jenkins/` folder. The jenkinsfile under jenkins/ can be used for a multibranch jenkins project, which builds
```
Comment (twicki) [inline tmpl/README.md.j2:89]: We need to change the two to be one

**Review by twicki — CHANGES_REQUESTED** [review]

File: tmpl/jenkins/Jenkinsfile.j2 (hunk 1 above)
Comment (cosunae) [reply tmpl/jenkins/Jenkinsfile.j2:35]: yes, I always thinking about it before removing. but then I thought that the assumption is that at the beginning of each stage we are always in the WORKSPACE of the jenkins plan, and so it was unnecessary

File: tmpl/README.md.j2 (hunk 2 above)
Comment (cosunae) [reply tmpl/README.md.j2]: done, thanks for your commits

**Pull request merged**